src/App.tsx

```diff
@@ -1,26 +1,43 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useDispatch } from 'react-redux';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { todosSlice } from './features/todos';
+import { useAppSelector } from './app/hooks';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [loading, setLoading] = useState(false);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useDispatch();
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    setLoading(true);
+    getTodos()
+      .then((data: Todo[]) => dispatch(todosSlice.actions.setTodos(data)))
```

Comment:
better use async actions for fetching data
read more https://redux-toolkit.js.org/api/createAsyncThunk